onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -427,10 +379,10 @@ SubGraphCollection_t TensorrtExecutionProvider::GetSupportedList(SubGraphCollect
       if (group.second) {
         nodes_list_output.push_back(group);
       } else {
-        onnxruntime::Model model_build(graph.Name(), true, ModelMetaData(), PathString(),
-                                       IOnnxRuntimeOpSchemaRegistryList(), graph.DomainToVersionMap(),
-                                       std::vector<ONNX_NAMESPACE::FunctionProto>(), *GetLogger());
-        onnxruntime::Graph& graph_build = model_build.MainGraph();
+        std::unique_ptr<onnxruntime::Model> model_build = std::make_unique<onnxruntime::Model>(graph.Name(), true, ModelMetaData(), PathString(),
```

Comment:
we can't use std::make_unique() as we're supposed to be c++11 compliant.
that's why we have onnxruntime::make_unique()